deploy/helm/prometheus-overrides.yaml

```diff
@@ -34,10 +34,6 @@ nodeExporter:
     interval:
 # Ensure we use pre 1.14 recording rules consistently as current content depends on them.
 kubeTargetVersionOverride: 1.13.0-0
-## Set the enabled flag to false for either of the below two purposes:
-## 1. Not install prometheus operator helm chart as a dependency along with this helm chart
-## 2. Disable metrics collection altogether
-enabled: true
```

Comment:
I think this should stay if someone would like to disable `prometheus-operator` only (in case they use their own prometheus but still want to use fluentd for tagging)